src/Components/NotificationDropdown.js

```diff
@@ -0,0 +1,108 @@
+import React, { Fragment, Component } from "react";
+import { withRouter } from 'react-router';
+import { MDBDropdown, MDBDropdownToggle, MDBDropdownMenu, MDBDropdownItem } from "mdbreact";
+import axios from "axios";
+import './styles/NotificationDropdown.css'
+
+function NotifIcon(props) {
+    const notifType = props.value._notification_type;
+    const isRead = props.value.is_read ? 'var(--medgray)' : 'var(--saturatedpurple)';
```

Comment:
Add on I missed, I would call this `commentColour` or something of the sort because that infers what the value type is. Whereas `isRead` makes me think it's a boolean.